src/calculateRentalCost.js

```diff
@@ -22,13 +23,29 @@
  *  - calculateRentalCost(3) === 100
  *  - calculateRentalCost(7) === 230
  *
- * @param {number} days
+ * @param {number} payTotal
  *
  * @return {number}
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const payOneDay = 40;
+  const discountMore3Days = 20;
+  const discountMore7Days = 50;
+
+  const payTotal = days * payOneDay;
+  const rentMore3Days = payTotal - discountMore3Days;
+  const rentMore7Days = payTotal - discountMore7Days;
+
+  if (days >= 7) {
```

Comment:
This is not as crucial as suggestion above, but better move amount of days into const too, like bigDiscountThreshold, for example